src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Node<T> {
+    private T value;
+    private Node<T> next;
+    private Node<T> prev;
+
+    public Node(Node<T> prev, T value, Node<T> next) {
+        this.value = value;
+        this.next = next;
+        this.prev = prev;
+    }
+
+    public T getValue() {
+        return value;
+    }
+
+    public void setValue(T value) {
+        this.value = value;
+    }
+
+    public Node<T> getNext() {
+        return next;
+    }
+
+    public void setNext(Node<T> next) {
+        this.next = next;
+    }
+
+    public Node<T> getPrev() {
+        return prev;
+    }
+
+    public void setPrev(Node<T> prev) {
+        this.prev = prev;
+    }
+}

```

Comment:
The Node class should be an inner class inside your LinkedList class. This encapsulates the Node class within the LinkedList class, which is the only place it should be used.